gateway/conf/nginx.conf.liquid

```diff
@@ -77,6 +77,9 @@ http {
     {% include tracer_conf %}
   {% endif %}
 
+  ssl_session_fetch_by_lua_block { require('apicast.executor'):ssl_session_fetch() }
```

Comment:
I don't see `ssl_session_fetch()` and `ssl_session_store()` defined anywhere. What I am missing?

Reply:
Actually I did not want to commit those lines just say. Will remove them. Want to keep it for the future.